### PR TITLE
Update ecosystem-user-event.mdx

### PR DESCRIPTION
**Pull request by claidler**
Typo fix

### CODE REVIEW

File: docs/ecosystem-user-event.mdx [hunk 1]
```diff
@@ -49,7 +49,7 @@ in
 
 ### `click(element, eventInit, options)`
 
-Clicks `element`, depending on what `element` is it can have different side
+Clicks `element`, depending on what `element` it is, it can have different side
```
Comment (eps1lon) [inline docs/ecosystem-user-event.mdx:52]:
```suggestion
Clicks `element`, depending on what `element` is clicked, calling `click()` can have different side
```

Really hard to follow what "it" is referring to

Comment (claidler) [reply docs/ecosystem-user-event.mdx:52]: Agreed 👍🏻

**Review by eps1lon — APPROVED** [review]
Thanks!

**Comment by eps1lon** [comment]
@all-contributors Add @claidler for docs
